src/Graphics/src/Graphics/Platforms/Android/PlatformImage.cs

```diff
@@ -156,8 +156,16 @@ public IImage ToPlatformImage()
 
 		public static IImage FromStream(Stream stream, ImageFormat formatHint = ImageFormat.Png)
 		{
-			var bitmap = BitmapFactory.DecodeStream(stream);
```

Comment:
Do we know why the read is not completed before the call ends?

Is it a delayed read? If so, can we call some property or method to get bounds or something to trigger the read instead of copying? What if the image is 30MB? That will be a lot of duplicate memory in an expensive area.

Reply:
Updated the implementation, can you review again?